README.md

```diff
@@ -1 +1,238 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello-world-microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a starting project to implement Linters and to create new branches.
+
+## 🛠 Built With <a name="built-with">VS CODE EDITOR</a>
+
+### Tech Stack <a name="tech-stack">HTML & CSS</a> 
+  <details>
+    <summary>Client</summary>
+    <ul>
+      <li>HTML</li>
+      <li>CSS</li>
+    </ul>
+  </details>
+
+### Key Features <a name="key-features"></a> 
+
+Displays Hello-Microverse text on web page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to clone github repository using command : git clone https://github.com/PremChand12/hello-world-microverse
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder: cd my-folder
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with: cd my-project
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, use code editor e.g Visual Studio
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, use Visual Studio Live Server
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using commands:
+git add,
+git commit and,
+git push
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Prem Chand**
+
+- GitHub: [@githubhandle](https://github.com/PremChand12)
+- Twitter: [@twitterhandle](https://twitter.com/Chandu1111112)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/prem-chand-maddati-15a69a1a8/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
Please kindly add features that will be implemented in the future,so a user knows what to be expected in the future 😄